CONTRIBUTING.md

```diff
@@ -1,25 +1,47 @@
+---
+editor: 
+  markdown: 
+    wrap: 72
+---
+
 # Contributing to the Analyst's Guide
 
-Thank you for investing your time in contributing to the analyst's guide! All contributions are very welcome - we want this to be a useful resource for the whole analytical community in DfE :sparkles:.
+Thank you for investing your time in contributing to the analyst's
+guide! All contributions are very welcome - we want this to be a useful
+resource for the whole analytical community in DfE :sparkles:.
```

Comment:
Did you tidy these line lengths manually or was there a package you used to do it?